department-of-veterans-affairs/caseflow
# Add CAVC Remand Section to Case Details

Resolves #15227

### Description
Adds the CAVC Remand sections to the case details view of CAVC ama appeals.

### Acceptance criteria
- [ ] Verify new section is created in Case Details called CAVC Remand that displays the following information stored via the court order entry form: 
  - Court docket: {Number}
  - Attorney representation: {Yes/No}
  - Name of Judge: 
  - Procedure: 
  - Type: {JMR or JMPR}
  - Court's decision date: {Date} 
  - Judgement date: {Date}
  - Mandate date: {Date} 
- [ ] Verify the stored information is displayed on the new appeal's case details page
- [ ] Verify CAVC Remand is displayed in side navigation
- [ ] Include screenshot(s) in the Github issue if there are front-end changes

### Testing Plan
1. Login as BVAAABSHIRE
2. Search for a CAVC appeal (should have one in our seed data under docket #201027-892)
3. Ensure that CAVC Remand is listed in the SideNav section of the page
4. Make sure all eight sections listed in the AC are present and are correctly populated & formatted
5. Check both a non-CAVC ama appeal and a legacy appeal to make sure neither show the CAVC Remand section

### User Facing Changes

#### BEFORE
<img width="1245" alt="Screen Shot 2020-10-30 at 12 32 12 AM" src="https://user-images.githubusercontent.com/18618189/97660305-83225e80-1a47-11eb-9410-78446dae1c3c.png">

#### AFTER
<img width="1240" alt="Screen Shot 2020-10-30 at 12 32 29 AM" src="https://user-images.githubusercontent.com/18618189/97660327-97fef200-1a47-11eb-914e-4b8a86068c2a.png">

### Storybook Story
*For Frontend (Presentationa) Components*
